src/content/docs/apis/intro-apis/new-relic-api-keys.mdx

```diff
@@ -53,12 +47,16 @@ You can view and manage most API keys from the API keys UI page, which is at [on
   From [one.newrelic.com/launcher/api-keys-ui.api-keys-launcher](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher), you can create and manage license keys, browser keys, and user keys.
 </figcaption>
 
-Other ways to create and manage keys:
+You must be logged in to your New Relic account to use API keys. If you haven't already, [sign up for free](https://newrelic.com/signup) so you can start observing your data right away!
```

Comment:
Going back and forth here. I'm not convinced we need the first sentence and it might be better to have a clear CTA without additional preface text... (Prob fine to leave as is though)

Reply:
I kind of waffle on this right now too. I'll leave the conversation open in case B or A have any thoughts on it.

Reply:
A couple thoughts:

* I'm a little worried about losing the prominent product link near top of page vs the image caption
* Not sure we need the explicit signup link here

Reply:
Agreed, I'm going to rework this into another getting started 1-2-3 like we did for Applied Intelligence and solve everyone's problems in one go I think. Thank you guys for the feedback!